### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by mikez321**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling (Styling to new or existing files)
- [ ] Testing
### Detailed Description
This PR will create BE migrations and set up the DB.  **It does not have any additional tests yet!

### Why is this change required? What problem does it solve?
This change will allow the input of plants into the DB.

### Was any previously written code changed?
Nope

### Directory modified:
We created migrations using alembic.

### CODE REVIEW

File: alembic/versions/8c089a511ab7_create_plants.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+    'plants',
+        sa.Column('id', sa.Integer, primary_key=True),
+        sa.Column('name', sa.String(50)),
+        sa.Column('plant_type', sa.String(50)),
+        sa.Column('image', sa.Unicode(255)),
```
Comment (kelshadarby) [inline alembic/versions/8c089a511ab7_create_plants.py:25]: We're assuming this means textfield, once we begin seeding the db we should be able to tell if this needs to be changed

File: alembic.ini [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# A generic, single database configuration.
```
Comment (kelshadarby) [inline alembic.ini:1]: We generated this root alembic.ini so that we can run migrations and seed the db using alembic CLI

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
-  - pip install -r requirements.txt
-# command to run tests
+- pip install -r requirements.txt
 script:
-  - pip install flask_api
-  - pip install flask-sqlalchemy psycopg2 flask-migrate
-  - pytest
+- pip install flask_api
+- pip install flask-sqlalchemy psycopg2 flask-migrate
+- pytest
```
Comment (kelshadarby) [inline .travis.yml:9]: We should try to get all of our tests inside a `test` folder and see if pytest will still

File: alembic/versions/8c089a511ab7_create_plants.py (hunk 1 above)
Comment (mikez321) [reply alembic/versions/8c089a511ab7_create_plants.py:25]: If it doesn't mean that, we'll get to practice more migrations!

**Pull request merged**